colorcoder/Main.java

```diff
@@ -1,120 +1,57 @@
-package colorcoder;
```

Comment:
same name (Main) used for Class and Method !
Main class holds unrelated responsibilities